__tests__/FunctionTest.js

```diff
@@ -0,0 +1,12 @@
+import 'react-native';
+import React from 'react';
+import Home from '../src/Home';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Airbenders-490/AirbendersFE/pullRequest?prid=8247177)
```suggestion
import Home from "../src/Home";
```